app/services/notifications/v5m0/MessageDetails.scala

```diff
@@ -43,7 +43,7 @@ object MessageDetails {
-      case NewRenewalReminder => s"<p>It is time to pay your business’s annual fee.</p><p>You need to check and verify that your details are up to date and pay your annual fee in full by ${ formatDate(endDate) }.</p><p>If you do not pay your fees by the above date, HMRC will cancel your business’s registration.</p><p>Start this process from your status page. Completing this will tell you the amount of your annual fee.</p><p>HMRCs money laundering registration policy will assist you in understanding and calculating your fee.</p><p>Details of how to pay your fee can be found on www.gov.uk.</p><p>If you are experiencing difficulties in carrying out these actions, please contact MLRCIT@hmrc.gov.uk.</p><p>If your business is not registered with a relevant supervisory body (such as HMRC) you and/or your business may be subject to civil sanctions, such as a fine, or criminal proceedings if you continue to trade in activities covered by the Regulations.</p><p>If you have taken the steps outlined above, please ignore this message.</p>"
+      case NewRenewalReminder => s"<p>It is time to pay your business’s annual fee.</p><p>You need to check and verify that your details are up to date and pay your annual fee in full by ${ formatDate(endDate) }.</p><p>If you do not pay your fees by the above date, HMRC will cancel your business’s registration.</p><p>Start this process from your status page. Completing this will tell you the amount of your annual fee.</p><p>HMRCs money laundering registration policy will assist you in understanding and calculating your fee.</p><p>Details of how to pay your fee can be found on <a href=https://www.gov.uk>www.gov.uk.</a></p><p>If you are experiencing difficulties in carrying out these actions, please contact MLRCIT@hmrc.gov.uk.</p><p>If your business is not registered with a relevant supervisory body (such as HMRC) you and/or your business may be subject to civil sanctions, such as a fine, or criminal proceedings if you continue to trade in activities covered by the Regulations.</p><p>If you have taken the steps outlined above, please ignore this message.</p>"
```

Comment:
Should href=https://www.gov.uk>www.gov.uk. not have quotation marks e.g. 
href="https://www.gov.uk>www.gov.uk."

Reply:
whoop, i have to fix test case as well

Reply:
OK great just wanted to check.